bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java

```diff
@@ -183,7 +182,9 @@ public boolean exists(Get get) throws IOException {
       for (Get get : gets) {
         existGets.add(GetAdapter.setCheckExistenceOnly(get));
       }
-      return getBatchExecutor().exists(existGets);
+      try (BatchExecutor executor = createBatchExecutor()) {
```

Comment:
Will it be expensive if everytime a method is called it'll create and close a batcher?

Reply:
I dont think so. It just allocates a handful of objects. I dont think it will be noticeable